_docs/2017-01-01-index.md

```diff
@@ -151,9 +151,11 @@ which is a worthy read for anyone with a background in cryptocurrency
 theory. The core idea of Proof of Stake is that instead of wasting
 electricity on cracking computationally heavy problems, a node is
 selected to mint a new block, with a probability proportional
-to the amount of coins a particular node has. A node that attempts to
-generate a block is called _a slotleader_. You can read more about
-this process in [Proof of Stake in Cardano SL](/cardano/proof-of-stake/).
+to the amount of coins a particular node has. Every node of network can attempt to
```

Comment:
Well, node can make an attempt, but there is no reason to do it unless you are a slot leader :/